sonar-batch/src/main/java/org/sonar/batch/issue/DeprecatedIssueWrapper.java

```diff
@@ -81,8 +81,17 @@ public Integer line() {
     return textRange != null ? textRange.start().line() : null;
   }
 
+  /**
+   * @deprecated since 5.5, replaced by {@link #gap()}
+   */
   @Override
+  @Deprecated
   public Double effortToFix() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)